### PR TITLE
Consolidate test suites

### PR DESCRIPTION
**Pull request by ycao56**
The 1st commit moved policy-collection test cases into integration.
The 2nd commit added label to all test cases and updated the run scripts. 
https://github.com/stolostron/backlog/issues/20815

Signed-off-by: Yu Cao <ycao@redhat.com>

### CODE REVIEW

**Comment by mprahl** [comment]
@ycao56 the diff is quite large so it's hard to review. Could you please separate this out into two commits? The first commit would be moving the files from policy-collection to integration and the second commit would be the changes made to the test files to make them work.

**Comment by ycao56** [comment]
> @ycao56 the diff is quite large so it's hard to review. Could you please separate this out into two commits? The first commit would be moving the files from policy-collection to integration and the second commit would be the changes made to the test files to make them work.

Sure. I will update.

**Comment by ycao56** [comment]
@mprahl this is ready for review.

File: build/run-e2e-tests-policy-framework-prow.sh [hunk 1]
```diff
@@ -31,11 +31,18 @@ $DIR/patch-cluster-prow.sh
...
+  echo "* No GINKGO_LABEL_FILTER set"
+else
+  GINKGO_LABEL_FILTER="--label-filter=${GINKGO_LABEL_FILTER}"
+  echo "* Using GINKGO_LABEL_FILTER=${GINKGO_LABEL_FILTER}"
+fi
+
 echo "===== E2E Test ====="
 echo "* Launching grc policy framework test"
-for TEST_SUITE in policy-collection integration; do
+for TEST_SUITE in integration; do
```
Comment (mprahl) [inline build/run-e2e-tests-policy-framework-prow.sh:43]: Could you please remove the for loop?

**Review by mprahl — CHANGES_REQUESTED** [review]
One small comment, otherwise, this looks good!

File: build/run-e2e-tests-policy-framework-prow.sh (hunk 1 above)
Comment (ycao56) [reply build/run-e2e-tests-policy-framework-prow.sh]: done

**Review by mprahl — APPROVED** [review]

**Pull request merged**